devops/iam-did-auth-proxy-dev/values.yaml

```diff
@@ -113,7 +113,7 @@ did-auth-proxy-helm:
     enabled: true
     annotations: {}
     encryptedData:
-      IPFS_PROJECTSECRET: AgAIpovKHlyFYxOT6Nk4d+T6m9zKF95ZdUgQPx/fsWZw1aAQJDXzDL9CZBI++zZ4pXnra4L4EftL8hyJwKPqSSVp9IjB+IcOb2wC3EaE4IEiMl2HGbzpqKtrmMoik4NkPbC1N7pwowuO43tttE2eUZblz85+3g+ccCTz/GouzGY/ryBKpVADGEafrrbtv6X41NsGYpe+Pzu6fApjIP9gJzkdwW8nIo8l5sUqfSZHkIKlGvJuuIp29l5IKeQyQemvHk0pD4/MAOmzYoe8my6ZO/6c9l4SBnbtG7BqrIqhKlRN4hw3RINVC6bt6dkSuznlZRos24pAKOufPoRcZvdIUti2a5KheXLCVxO0l99sBOlW674mT2Qim+jdORNxAJbt3pUhy/xHwWxxBWn++J7WAgvOYpihSd/bKsKm2JcMlvHKvjFY2hGzjPlrAP38BiJKgC4SMIA3DAouylt/6y8U9Fwd2d4GnPCsKPewBvqSCdxMgT331tfV/BeTaHV70WcE02k8qX1xOrFHkmiaSQo1kimh9KCe/p9gB1EuGeN8Z3bbUbcAd8n4P4Ry0pIUaaF/VNqi4eM3+Kpgx8aS5fa8o9iKyYVKMPNTzIKEPYp63u+TQmOUX1Kj8M/67ZqItB0XiHyocVFcTdDfITrGPjilxwyxZJrrNWWIiqUFCfSJhZelBnQ+qrGrHOUNdmq29uHYKf2w5MoXwGXmeSLNbiur9QghKtl3QJPDQFTTn2Ulv/aThQ==
+      IPFS_PROJECTSECRET: AgCnApFbM7NTtWcL+CUo4WadFhRAdlRqNIo57JRtvRE5++IdJhh9MBZ+nKrpI+TO9e1eWXMbjjBKiAnlIZLd5D5Gn2KGTdeMl0e3ZOZKnEkxU6US5rajcMQ5tvzHU0uy/pEY6r/pAzhl40NvhGlPyTY5USaZRevki8ns/A0p5xkdNq9HUO/SnTIJ9n3TjoTj6UD6LyT4EGJKhX96ZswcKJTPiM7SOq3m5ZMr+HlgOCKyNDOPEfS0SXn0Yb0gugyrET6C4pmTFTi1UjcEX5FLMBMIIN6Y514xyx/s8fC6ioPGd5SkT6GzAOBbd99Vem+rJQD7c+ks4UOBEBRvitnlH8SyQIbUasvl8va7lzSyPbgwM5oDOBjk2AYXPKbQ9qo2w5nDro/4MYY5QM0+uP4snMkbZVa2QXRJNxCkFWt/bgdBV6Z8rc0PZ0QzxDtpaKoh3f7UU9rETGaVDoU8TQv7QtsD6D7h/Fh66xlmb4oGQNhuIhqU/+aMJWpy9CuHsklpKpQehWaCTl/PpKanRNHQ9KRg+Qu5eGHLpGbXE7V6jtzhNsiGUa1GIH0cZlWQQS5qkDzDraOoLbabuZ/Erxtux5fT6XKcUBaxWTG4wD75qncJ+mAhjviwlT9wSGrTvzDXSRyREmVKaDEKRDtLtH32vA+fifZAGh9qWjdZVF6lOml/2gLorn9++dzN+fo+Lk0zlsUrZ+rGmoEnROePkls1+15b6NPgovuPRMu/LA9ql3Kjow==
```

Comment:
This is because project id has been changed from [2MdfPOCDrxelBHhNLArgfuG7pmB](https://github.com/energywebfoundation/did-auth-proxy/blob/e382f5bd5e46fb657935d4553b29c0119ebd4577/devops/iam-did-auth-proxy-dev/values.yaml#LL80C21-L80C48) on gp4btc-stg to [2DXJGR6JFXIE612O8bUvjXGIBIP](https://github.com/energywebfoundation/did-auth-proxy/blob/c15701dcb4d0b6fc3fbfcf2eb4aeda0fbd47bda3/devops/iam-did-auth-proxy-dev/values.yaml#LL84C21-L84C48) on develop
I think that `2DXJGR6JFXIE612O8bUvjXGIBIP` is correct value, because same is used in dev [environment](https://github.com/energywebfoundation/did-auth-proxy/blob/c15701dcb4d0b6fc3fbfcf2eb4aeda0fbd47bda3/devops/dev/values.yaml#L108)

Reply:
Hi @JGiter , can you explain this point a bit more please?
> This is because project id has been changed from 2MdfPOCDrxelBHhNLArgfuG7pmB on gp4btc-stg to 2DXJGR6JFXIE612O8bUvjXGIBIP on develop

The `IPFS_PROJECTID` in this PR is `2DXJGR6JFXIE612O8bUvjXGIBIP`. I assume that the new value for `IPFS_PROJECTSECRET` that is suggested in this PR corresponds to `2MdfPOCDrxelBHhNLArgfuG7pmB`. So, if we merge the PR as is, I think the `IPFS_PROJECTID` and `IPFS_PROJECTSECRET` would not match.

Reply:
May be you are right. I decided that `2DXJGR6JFXIE612O8bUvjXGIBIP` corresponds to `AgCnApFbM7NTtWcL+`, because same values are in dev [environment](https://github.com/energywebfoundation/did-auth-proxy/blob/develop/devops/dev/values.yaml). How can you see that secret `AgCnApFbM7NTtWcL+` corresponds to project `2MdfPOCDrxelBHhNLArgfuG7pmB`? In prod `2MdfPOCDrxelBHhNLArgfuG7pmB` matches to `AgCjYhRFNMPlVlMB09bEhWERAYbB+` https://github.com/energywebfoundation/did-auth-proxy/blob/256fc53ab7e435681a3d1b0b885f441ea21e0f51/devops/iam-did-auth-proxy-prod/values.yaml#LL114C27-L114C55

Reply:
@JGiter I see your point about `AgCnApFbM7NTtWcL+` corresponding to `2DXJGR6JFXIE612O8bUvjXGIBIP` for the "dev" environment. I double-checked the current value that is decrypted from `AgAIpovKHlyFYxOT6Nk4d+` in the "iam dev" deployment of did-auth-proxy (which is the deployment that is affected by this values.yaml file), and the expected value is decrypted (as expected based on the value in the EW secrets manager).

I think that it would also be strange if this is the one change that was correctly made on `gp4btc-stg` branch but somehow wasn't copied to `dev`

As such, I think we should not execute this secret change and we should keep `AgAIpovKHlyFYxOT6Nk4d+` encrypted value. If you still believe that `AgCnApFbM7NTtWcL+` is correct, we can merge and see how it is, no problem.

Reply:
Alright, then this secret just differently encoded. I reverted changing of secret